Composer/packages/extensions/extension/src/hooks/useLgApi.ts

```diff
@@ -32,18 +32,18 @@ export const useLgApi = (shellApi: ShellApi) => {
     return targetTemplate ? targetTemplate.body : lgText;
   };
 
-  const createLgTemplate = async (
```

Comment:
why all asycn keywords were removed?

Reply:
yes, I removed all async functions

Reply:
@lei9444 the reason we were able to remove all the async is because we are using the new botbuilder-lg and lu libs correct?